baseline/OSPS-QA.yaml

```diff
@@ -545,3 +545,36 @@ controls:
           primary branch. This can be achieved by requiring a pull request to be
           reviewed and approved by at least one other contributor before it can
           be merged.
+  - id: OSPS-QA-08
+    title: |
+      The project MUST store and manage all secrets and credentials used by the project in a secure manner.
+    objective: |
+      Ensure that sensitive data is not disclosed, compromised or misused leading to security vulnerabilities or data breaches.
+    guideline-mappings:
+      - reference-id: BPB
+        identifiers:
+          - S-B-5 # TODO: is this the right numbering for https://www.bestpractices.dev/en/criteria#0.no_leaked_credentials
+      - reference-id: SSDF
+        identifiers:
+          - PO.1.1
+          - P0.3.1
+          - P0.4.2
+          - PO.5.1
+          - PW.1.2
+          - PW.1.3
+          - PW.5.1
+    assessment-requirements:
+      - id: OSPS-QA-08.01
+        text: |
+          The project MUST define a policy for managing secrets and credentials used by the project. The policy should include guidelines for storing, accessing, and rotating secrets and credentials.
```

Comment:
Do we need a caveat for any situations where the project does not manage its own secrets? 

For example, some FINOS projects don't have anything that requires secrets. And in cases where cloud resources are used in the test pipeline, FINOS manages cloud credentials outside of the project's scope.